File: Strings/3141/3141.c
#include <stdio.h>

int main(){
    char nome[51];
    int da, ma, aa, dn, mn, an;
    gets(nome);
    scanf("%d/%d/%d", &da, &ma, &aa);
    scanf("%d/%d/%d", &dn, &mn, &an);
    if (da == dn && ma == mn){
        printf("Feliz aniversario!\n");
        printf("Voce tem %d anos %s.\n", aa-an, nome);
    }
    else if (ma > mn) printf("Voce tem %d anos %s.\n", aa-an, nome);
    else if (ma == mn && da>dn) printf("Voce tem %d anos %s.\n", aa-an, nome);
    else printf("Voce tem %d anos %s.\n", aa-an-1, nome);
    return 0;
}

File: Strings/3141/3141.py
nome = input()
da, ma, aa = map(int, input().split("/"))
dn, mn, an = map(int, input().split("/"))
if da == dn and ma == mn:
    print("Feliz aniversario!")
    print(f"Voce tem {aa-an} anos {nome}.")
elif ma > mn:
    print(f"Voce tem {aa-an} anos {nome}.")
elif ma == mn and da>dn:
    print(f"Voce tem {aa - an} anos {nome}.")
else:
    print(f"Voce tem {aa - an - 1} anos {nome}.")